bzt/utils.py

```diff
@@ -1019,3 +1020,12 @@ def humanize_bytes(byteval):
     # (.4g results in rounded numbers for exact matches and max 3 decimals,
     # should never resort to exponent values)
     return '{:.4g}{}'.format(byteval / (1 << (order * 10)), _suffixes[order])
+
+
+def replace_leading_tabs(text, spaces=4):
+    replacement = ' '*spaces
+    replaced_lines = []
+    regex = re.compile('^ *(\t+) *')
```

Comment:
Maybe have multiline regexp that will capture all `\n\t+` and then just replace found combinations in string?